package.json

```diff
@@ -2,6 +2,7 @@
   "name": "shopify-app-extension-template",
   "version": "0.1.0",
   "main": "extension.ts",
+  "engines" : { "node" : "^10.12.0 || >=12.0.0" },
```

Comment:
I think doing the same as Argo Checkout works https://github.com/Shopify/argo-checkout-template/blob/a182fd6406a8d4ac0f03c3212daea34dbfe3d4c9/package.json#L5.

* Node doesn't have v11
* Argo Checkout specifies >= 10.16.0
* `"node": "^10.12.0 || >=12.0.0"` So, if people have 10.16.0 it will match anyway.